README.md

```diff
@@ -9,13 +9,20 @@ The projection methods implemented are: Discriminant Analysis Tensor Representat
 ## Direct classification methods
 
 # Dependencies
-ManOpt for optimisation over manifolds in Matlab
 
-Tensor manipulation code by Morten Mørup, available at http://www.imm.dtu.dk/~mm/downloads/CPandTucker.zip
+## Matlab
 
-immoptibox http://www2.imm.dtu.dk/projects/immoptibox/
+* ManOpt for optimisation over manifolds in Matlab
 
-Code assumes Matalb R2018b or higher
+* Tensor manipulation code by Morten Mørup, available at http://www.imm.dtu.dk/~mm/downloads/CPandTucker.zip
```

Comment:
Should we have periods at the end of list items?

Reply:
I don't believe it is necessary for comprehension of the list.

Reply:
Sounds good, we can always add in the future if we decide for that style